tendermint/src/rpc/error.rs

```diff
@@ -95,12 +96,6 @@ impl Display for Error {
     }
 }
 
-impl Fail for Error {
-    fn name(&self) -> Option<&str> {
-        self.code.name()
-    }
-}
-
```

Comment:
Had to remove this as `std::error::Error` implements this too. I more convinced now that we should switch to thiserror + anomaly everwhere in this repo.
ref  #150

Reply:
The KMS doesn't use the RPC client (yet) so that should be fine

Reply:
Cool, thanks!

Reply:
I'll change the RPC errors to also use thiserror + anomaly in a different PR then.